mkosi/__init__.py

```diff
@@ -1163,15 +983,18 @@ def calculate_bmap(state: MkosiState) -> None:
         run(cmdline)
 
 
+def acl_allow_remove(root: Path, uid: int) -> None:
+    run(["setfacl", "-R", "-m", f"user:{uid}:rwx", root,
```

Comment:
By default a recursive setfacl will follow symlink arguments, but not symlinks it encounters later. If you can't think of a good reason against it, I'd rather throw in a `--physical` here just to be on the safe side.

Also, we should probably be lenient here, if this fails and just emit a warning, that people will have to clean up their stuff with elevated privileges, since I'm sure somebody will try this on an FS that can't do ACLs or only has them as opt-in (i.e. zfs).